src/metabase/integrations/ldap.clj

```diff
@@ -8,68 +8,69 @@
              [setting :as setting :refer [defsetting]]
              [user :as user :refer [User]]]
             [metabase.util :as u]
+            [puppetlabs.i18n.core :as i18n :refer [tru]]
```

Comment:
No need to do `:as i18n` here because you aren't using `i18n` for anything. If we did something like `(i18n/tru ...)` this would make sense but we're not